index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href="style.css" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio setup and mobile</title>
+</head>
+<body>
+       
+      
+       <section class="headline">
+       
+        <header class="toolbar">
+            <a href="#" ><p>My logo </p></a>
+            <div class ="vector">
+                <a href="#" >   <img src = "/Icons/Union.png"> 
+                </a>
+            </div>
+              
+        </header>
+   
+
+        <div class="header"><h1>I’m Mariane Glad to see you!</h1></div>
+        <div class="info"><h3>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h3></div>
+        <div class="connect"><h3>Lets connect</h3></div>
+        
```

Comment:
- [ ] Please remove unnecessary divs as you are already using HTML semantic tags here. You should keep the classes. 
- [ ] Your class 'info' should be inside the p tag for semantic reasons.